lib/fluent/plugin/storage_local.rb

```diff
@@ -64,7 +64,8 @@ def configure(conf)
             end
           end
         elsif root_dir = owner.plugin_root_dir
-          @path = File.join(root_dir, 'storage.json')
+          basename = (conf.arg && !conf.arg.empty?) ? "storage.#{conf.arg}.json" : "storage.json"
```

Comment:
Ah, I got it.
We should fill up `usage` when conf is `Fluent::Config::Element`'s instance case: https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin_helper/storage.rb#L56